core/src/main/resources/db/migration/V10__add_update_spec_install_pos.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE UpdateRequest
```

Comment:
Ok, this makes sense for a small number of vehicles but I wonder how do we transpose this for a large fleet...